react-native/react-native_2.md

```diff
@@ -0,0 +1,31 @@
+# About React and React Native components
+
+React Native requires you to have an index.[os].js for each platform. But any other component can be written once for both OSes.
+
+Components are written *the React way*:
+
+  * A component is a Javascript class (as introduced in ES6, but you can also write your code in a more traditional ES5 style).
+  * Each component is passed `props`, containing information useful for the component coming from its parent. For example, a text component will be passed the text to display and the styling of the text.
```

Comment:
Start by saying that each component has a state and a props, and then explain what each one is and what is their purpose